catalog-info.yml

```diff
@@ -1,11 +1,11 @@
 apiVersion: backstage.io/v1alpha1
 kind: Component
 metadata:
-  name: vercel-toolkit
+  name: vercel-nextjs-toolkit
```

Comment:
intentional name change, correct?  just making sure.  I see that this new name would match the repo name, and I'm not sure if there would be any implications if the two were out of sync anyway, but just figured I'd ask :)

Reply:
Yes, good callout! I am intentionally changing it to match the repo name since it was not appearing on roadie, and I believe that is because the repo name is not correct.